### ISSUE TITLE
[bitnami/cassandra] Regression: bitnami/cassandra fails to boot - ModuleNotFoundError: No module named 'six.moves'

### ISSUE THREAD

**Issue by tmoschou** [bitnami/containers#71313]
### Name and Version

bitnami/cassandra:4.1.6-debian-12-r2

### What architecture are you using?

arm64

### What steps will reproduce the bug?

1. docker compose up -d 
```yaml
services:
  cassandra:
    image: bitnami/cassandra:4.1.6-debian-12-r2
    ports:
      - '7000:7000'
      - '9042:9042'
    environment:
      - CASSANDRA_USER=cassandra
      - CASSANDRA_PASSWORD=cassandra
      - CASSANDRA_PASSWORD_SEEDER=yes
    volumes:
      - cassandra_data:/bitnami
volumes:
  cassandra_data:
    driver: local
```
2. tail container logs
3. wait for container startup to fail on timeout

### What is the expected behavior?

Container starts up normally

### What do you see instead?

Container times out could not connect
```
cassandra 02:28:37.50 INFO  ==>
cassandra 02:28:37.50 INFO  ==> Welcome to the Bitnami cassandra container
cassandra 02:28:37.50 INFO  ==> Subscribe to project updates by watching https://github.com/bitnami/containers
cassandra 02:28:37.50 INFO  ==> Submit issues and feature requests at https://github.com/bitnami/containers/issues
cassandra 02:28:37.51 INFO  ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit https://bitnami.com/enterprise
cassandra 02:28:37.51 INFO  ==>
cassandra 02:28:37.51 INFO  ==> ** Delaying Cassandra start by 10 seconds **
cassandra 02:28:47.51 INFO  ==> ** Starting Cassandra setup **
cassandra 02:28:47.52 WARN  ==> DB_HOST not set, defaulting to system hostname
cassandra 02:28:47.52 INFO  ==> Validating settings in DB_* env vars..
cassandra 02:28:47.52 WARN  ==> You set the environment variable DB_PASSWORD=cassandra. This is the default value when bootstrapping cassandra and should not be used in production environments.
cassandra 02:28:47.53 INFO  ==> Validating TLS settings in DB_* env vars..
cassandra 02:28:47.53 INFO  ==> Initializing cassandra database...
cassandra 02:28:47.66 INFO  ==> Deploying cassandra from scratch
cassandra 02:28:47.66 INFO  ==> Starting cassandra
cassandra 02:28:47.66 INFO  ==> Checking that it started up correctly
cassandra 02:29:47.68 INFO  ==> Found CQL startup log line
cassandra 02:29:48.40 INFO  ==> Nodetool reported the successful startup of cassandra
cassandra 02:29:48.40 INFO  ==> Password seeder node
cassandra 02:29:48.40 INFO  ==> Trying to access CQL server @ cassandra-4.1.6-r2
cassandra 02:46:38.22 ERROR ==> CQL command failed 100 times
cassandra 02:46:38.22 ERROR ==> Could not access CQL server
```

### Additional information

Running `cqlsh` inside the container returns
```
1001@cassandra-4:/$ cqlsh
Traceback (most recent call last):
  File "/opt/bitnami/cassandra/bin/cqlsh.py", line 134, in <module>
    from cassandra.cluster import Cluster
  File "/opt/bitnami/cassandra/bin/../lib/cassandra-driver-internal-only-3.25.0.zip/cassandra-driver-3.25.0/cassandra/cluster.py", line 33, in <module>
ModuleNotFoundError: No module named 'six.moves'
```

`bitnami/cassandra:4.1.6-debian-12-r1` was previously working

**Edit:** Looks like `cqlsh` bundled with apache-cassandra-4.1.6 doesn't work with Python 3.12
https://issues.apache.org/jira/browse/CASSANDRA-19206

**Comment by tmoschou** [comment]
Also related https://issues.apache.org/jira/browse/CASSANDRA-19245, the driver update wasn't back ported to 4.1
but Python 3.11 is the maximum supported version in 5.0-rc1 https://github.com/apache/cassandra/blob/cassandra-5.0-rc1/NEWS.txt#L207-L208

**Comment by dgomezleon** [comment]
Thanks for reporting.

**Comment by tmoschou** [comment]
Thanks @dgomezleon 

I can confirm fixed in [4.1.6-debian-12-r3](https://hub.docker.com/layers/bitnami/cassandra/4.1.6-debian-12-r3/images/sha256-aabe1eb600b6317fb91197c6912f6af40b3079f66d780d5cee579fa76208ecbd?context=explore)
